cypress/integration/page.spec.ts

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+/* eslint-disable max-len */
+import { faker } from '@faker-js/faker';
+import { peopleFromServer } from '../../src/data/people';
+import { Person } from '../../src/types/Person';
+
+enum DataQa {
+  Title = 'title',
+  SearchInput = 'search-input',
+  SuggestionsList = 'suggestions-list',
+  SuggestionItem = 'suggestion-item',
+  NoSuggestionsMessage = 'no-suggestions-message',
+}
+
+const getByDataQa = (id: DataQa) => cy.get(`[data-qa="${id}"]`);
+const getPageTitle = (person?: Person | null) => {
+  if (person) {
+    return `${person.name} (${person.born} - ${person.died})`;
+  }
+
+  return 'No selected person';
+};
+
+describe('Autocomplete page', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should display the initial page title', () => {
+    getByDataQa(DataQa.Title)
+      .should('contain', getPageTitle());
+  });
+
+  it('should display the input', () => {
+    getByDataQa(DataQa.SearchInput)
+      .should('be.visible')
+      .and('have.attr', 'placeholder', 'Enter a part of the name')
+      .and('have.value', '');
+  });
+
+  it('should not render "No matching suggestions" notification for empty input', () => {
+    getByDataQa(DataQa.NoSuggestionsMessage)
+      .should('not.exist');
+  });
+
+  it('should not display suggestions with empty focused input', () => {
+    getByDataQa(DataQa.SearchInput)
+      .focus();
+    getByDataQa(DataQa.SuggestionsList)
+      .should('not.be.visible');
+  });
+
+  it('should display "No matching suggestions" for non-matching input', () => {
+    getByDataQa(DataQa.SearchInput)
+      .type(faker.lorem.word());
```

Comment:
don't use random word here
For some reason it could become a part or some person name and the test will fail

please just hardcode it